src/setCompliment.js

```diff
@@ -0,0 +1,9 @@
+export default function setCompliment(arr1, arr2){
+  const concatSortArray = arr1.concat(arr2).sort()
+  const results = []
+  for (var i=0; i<concatSortArray.length; i++){
+    arr1.includes(concatSortArray[i])
+      ? null
+      : results.push(concatSortArray[i])
+  } return results
+}
```

Comment:
Good work. I reviewed alot your PR's. If you want, you could get a second pair of eye's. But otherwise it looks good to me. Maybe to get even more value you could do it a couple different ways and test the speed/efficiency.